test/integration/webhook/v1alpha2/resourceflavor_test.go

```diff
@@ -68,7 +69,7 @@ var _ = ginkgo.Describe("ResourceFlavor Webhook", func() {
 			resourceFlavor := testing.MakeResourceFlavor("resource-flavor").Label("foo", "@abcd").Obj()
 			err := k8sClient.Create(ctx, resourceFlavor)
 			gomega.Expect(err).Should(gomega.HaveOccurred())
-			gomega.Expect(errors.IsForbidden(err)).Should(gomega.BeTrue(), "error: %v", err)
+			gomega.Expect(err.Error()).To(gomega.ContainSubstring(errorInvalidLabelValue("@abcd")))
```

Comment:
revert

Reply:
The error returned is not of type forbidden. Hence, added the error substring. Any other way to assert the error?

Reply:
there should be an error type that matches in the package.

The substrings could change upstream, so I rather not pay attention to that.

Reply:
probably `errors.IsBadRequest`